src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1005,8 +941,12 @@ void Graph::PullOutputData(BlobMap &out) {
             IE_THROW(Unexpected) << "The CPU plugin graph doesn't contain output node with name: \"" << name << "\"";
         }
 
+        InferenceEngine::TensorDesc expectedDesc(
```

Comment:
yes, I agree. Actually we should remove all InferenceEngine class before 24.0 because it belong to legacy api.
This PR doesn't want to involve too much internal conversions and just mainly make interface api can be changed to new plugin api, so I think we can put such conversion into the following PR, which is not a small effort.

Reply:
Why can't we put the `aux_tensor` to the `out` map instead of the original tensor?